machAeroTutorials/overset_surface_mesh.rst

```diff
@@ -0,0 +1,608 @@
+.. _overset_surface_mesh:
+
+*********************************
+Surface Mesh
+*********************************
+
+Introduction
+============
+Now that we have a geometry, we can start meshing it. We are using Pointwise to generate the surface mesh. 
+This is not a full blown tutorial, more a walk through. If you want to learn more about it, I can highly 
+recommend their `youtube channel <https://www.youtube.com/user/CFDMeshing>`_\. You dont have to use 
+Pointwise to generate an overset mesh. ICEM or an other mesher would work aswell.
+
+Files
+=====
+Navigate to the directory ``overset/mesh`` in your tutorial folder. Either use the previously generated ``.igs`` file or copy it from the tutorial folder.
+::
+
+    $ cp tutorial/overset/geo/onera_m6.igs .
+
+It is possible to script Pointwise. In order to use it, we have to download the script first. You can either
+`download <https://raw.githubusercontent.com/pointwise/Semicircle/master/Semicircle.glf>`_ it here or copy it 
+from the tutorial folder.
+::
+
+    $ cp tutorial/overset/mesh/Semicircle.glf .
+
+Meshing strategy
+================
+Before we start meshing, we have to know how many meshes we create and where they overlap. For this tutorial,
+3 different meshes are proposed: ``near_wing``, ``near_tip`` and ``far``. The following picture should give an overview:
+
+.. figure:: images/overset_m6_mesh_fields.png
+    :width: 500
+    :align: center 
+
+    The three overset meshes.
```

Comment:
I find this figure not super clear, but I understand the challenge of visualizing the three separate meshes in 3D. 
Maybe you can add a 3D view of the wing with just the surface mesh (highlighting the different near field meshes) and a front/side 2D view to show the overlapping?